scripts/wrappers/dashboard_proxy.py

```diff
@@ -45,27 +78,13 @@ def dashboard_proxy():
         ]
         check_output(command)
 
-    print("Create token for accessing the dashboard")
-    run([KUBECTL, "apply", "-f", "-"], input=SECRET_YAML.encode("ascii"))
-
-    for attempt in range(20):
-        print("Waiting for secret token (attempt {})".format(attempt))
-        command = [
-            KUBECTL,
-            "-n",
-            "kube-system",
-            "get",
-            "secret",
-            "microk8s-dashboard-proxy-token",
-            "-o",
-            "jsonpath={.data.token}",
-        ]
-        output = check_output(command)
-        if output:
-            token = base64.b64decode(output).decode()
-            break
-
-        time.sleep(5)
+    # Get token created by the dashboard enable script
+    token = get_token("microk8s-dashboard-token")
+    if not token:
+        # Create a token. This is needed in case the enable script is pre-1.25.
+        print("Create token for accessing the dashboard")
+        run([KUBECTL, "apply", "-f", "-"], input=SECRET_YAML.encode("ascii"))
+        token = get_token("microk8s-dashboard-proxy-token")
```

Comment:
Since we are doing this either way, we don't have to use a separate name for the secret. Should we rename `microk8s-dashboard-proxy-token` to `microk8s-dashboard-token`?

Reply:
+1